### ISSUE TITLE
Table disappears with col_span in applet menu

### ISSUE THREAD

**Issue by shjohnson-pi** [linuxmint/cinnamon#4565]
Example applet that shows the issue: [Google Drive](https://drive.google.com/file/d/0B-wHLA1hDj9vOFZDZFVkS1Zsd28/view?usp=sharing). It's called "Table Test" and has an exclamation mark icon (`help-hint`).

Basically, if you have a table that has `x_fill: false`, and its only item is something that has a `col_span: 2`, the table will disappear. I'm guessing there is a problem in calculating the minimum width.

Here are some lovely ASCII art tables for a visual explanation.

```
+-|-|-+
|label|  This table (3x1) becomes hidden
+-|-|-+

+-|-|-+
|x|x|x|
|label|  This table (3x2) is visible, because other items do not have col_span
+-|-|-+
```

Please let me know if this is somehow the expected result. :)

**Computer Specs:**
- Ubuntu 15.04 x64
- Nvidia Proprietary drivers (355)
- Cinnamon 2.2.16

**Comment by shjohnson-pi** [comment]
Also occurs in Cinnamon 2.6.13, Linux Mint 17.2 x64

**Comment by Vahan86** [comment]
@shjohnson-pi, is this still an issue for you in Cinnamon 3.0?

**Comment by shjohnson-pi** [comment]
Yes, I still notice the problem on Cinnamon 3.0.7, Linux Mint 18 x64

**Comment by Vahan86** [comment]
@shjohnson-pi, is this still an issue in Cinnamon 4.4/4.6?

**Comment by shjohnson-pi** [comment]
I can confirm this is still an issue for Cinnamon 4.6.6 on Linux Mint 20 x64.
